doc/mgr/dashboard.rst

```diff
@@ -605,6 +605,36 @@ e.g. caused by certificates signed by unknown CA or not matching the host name::
 
 You can directly access Grafana Instance as well to monitor your cluster.
 
+Alternative URL for Browsers
+""""""""""""""""""""""""""""
+
+The Ceph Dashboard backend requires the URL to Grafana to be able to verify the
+existence of Grafana Dashboards before the frontend even loads them. Due to the
+nature of how Grafana is implemented in Ceph Dashboard, this means that two
+connections are required to successfully be able to see Grafana graphs in Ceph
+Dashboard::
+
+- The backend needs to verify the existence of the requested graph. If this
+  request succeeds, it lets the frontend know that it can safely access Grafana.
+- The frontend then requests the Grafana graphs directly from the users
+  browser using an iframe. The Grafana server is accessed directly without any
+  detour through Ceph Dashboard.
+
+Now, it might happen that your setup is somewhat difficult and the users browser
+cannot access the URL configured in Ceph Dashboard. To solve this issue, a
```

Comment:
```suggestion
Now, it might be the case that your environment makes it difficult for the user´s browser
to directly access the Grafana URL configured in Ceph Dashboard. To solve this issue, a
```